sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -17,6 +17,37 @@ Use the client library for App Configuration to create and manage application co
 - [App Configuration Store][app_config_store]
 
 ### Include the Package
+#### Include the BOM file
```

Comment:
Given that the outer header begins with `Include the` would it be better to have this section header be `Using azure-sdk-bom` and the one below being `Direct dependency`